hack/clean/clean.go

```diff
@@ -93,11 +93,13 @@ func run(ctx context.Context, log *logrus.Entry) error {
 	}
 
 	shouldDelete := func(resourceGroup mgmtfeatures.ResourceGroup, log *logrus.Entry) bool {
+		_, devCluster := resourceGroup.Tags["purge"]
```

Comment:
Now when I think about this I'm not sure if this is a good idea. If customer sets this (one or other way) we will not delete it instantly

Reply:
this is for the purger. this code should never run in customer subscription.